proposals/local-function-attributes.md

```diff
@@ -0,0 +1,37 @@
+# Attributes on local functions
+
+## Attributes
+
+Local function declarations are now permitted to have [attributes](https://github.com/dotnet/csharplang/blob/bb997294540bd02f939d60514634ccaa7abc659c/spec/attributes.md). Parameters and type parameters on local functions are also allowed to have attributes.
```

Comment:
These links should probably use relative paths instead of linking to the web at a particular commit.

Reply:
The relative path will be broken when this moves into the `csharp-9.0` directory :(

---
In reply to: [393936460](https://github.com/dotnet/csharplang/pull/3198#discussion_r393936460) [](ancestors = 393936460)